routes/index.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+var express = require('express');
+var router = express.Router();
+const mongoose = require('mongoose');
+const User = require('../modules/userdata');
+
+// BCrypt to encrypt passwords
+
+const bcrypt = require('bcrypt');
+const bcryptSalt = 10;
+
+// -- Connect to mongoose
+
+mongoose.Promise = Promise;
```

Comment:
Having this in app.js helps you to not repeat this code across all the routes files